packages/manager/apps/web/client/app/hosting/email/terminate/hosting-email-terminate.controller.js

```diff
@@ -0,0 +1,55 @@
+export default class HostingTerminateEmailCtrl {
+  /* @ngInject */
+  constructor($translate, Hosting) {
+    this.$translate = $translate;
+    this.Hosting = Hosting;
+  }
+
+  $onInit() {
+    this.fetchEmailOptions();
+  }
+
+  fetchEmailOptions() {
+    this.isLoading = true;
+    return this.Hosting.getEmailOptions(this.serviceName)
+      .then((options) => {
+        this.emailOptions = options;
```

Comment:
Is this redundant? :)

Reply:
Used to store both all options and store the first one.